.travis.yml

```
@@ -10,6 +10,14 @@ branches:
   only:
     - master
 
+matrix:
+  fast_finish: true
+  include:
+    - env: NAME="Docs"
+           MAIN_CMD="cd docs && make html"
+           PYTHON_VERSION=3.6
```

3.6 is already getting old...
```suggestion
           PYTHON_VERSION=3.7
```

---

Okay sure, we will switch to 3.7, but should deploy the docs off the 3.7 build then also. Ill fix it up.